router/handle.go

```diff
@@ -278,7 +278,7 @@ func (rt *Handle) commitStatusList(workerJobStatuses *[]workerJobStatus) {
 	transformedAtMap := make(map[string]string)
 	statusDetailsMap := make(map[string]*utilTypes.StatusDetail)
 	routerWorkspaceJobStatusCount := make(map[string]int)
-	var completedJobsList []*jobsdb.JobT
+	var rSourcesJobsList []workerJobStatus
```

Comment:
I don't believe we need this change, calling stats collector's methods in a for loop doesn't look to be simpler than calling it once for a batch of jobs & job statuses. Consistency between router & batchrouter code is another reason why we don't need this